src/pages/Swap/Components/ChartTabs.tsx

```diff
@@ -35,10 +37,10 @@ export const ChartTabs = ({
         {t('advancedTradingView.chartTabs.simple')}
       </Tab> */}
         <Tab
-          active={activeChartTab === ChartOptions.PRO}
+          active={activeChartTab === ChartOption.PRO}
           onClick={() => {
-            if (activeChartTab !== ChartOptions.PRO) {
-              setActiveChartTab(ChartOptions.PRO)
+            if (activeChartTab !== ChartOption.PRO) {
```

Comment:
should we add the setSelectedChartTab setter here as well when the pro mode is on?

Reply:
Yeah, this skipped in the merge. Fixed now!

Reply:
also It might be better solution to just have one getter and setter for activeChartTab instead of having one for localStorage and one with the context